src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToRfc3339Transformer.php

```diff
@@ -32,15 +39,25 @@ public function transform($dateTime)
         }
 
         if ($this->inputTimezone !== $this->outputTimezone) {
-            $dateTime = clone $dateTime;
+            if (!$dateTime instanceof \DateTimeImmutable) {
+                $dateTime = clone $dateTime;
+            }
```

Comment:
@xabbuh, I've no strong opinion on this. I just did it because I noticed `DateTimeImmutable` was not cloned in other transformers needing to clone the value, ref [`DateTimeToArrayTransformer`](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php#L80) and [`DateTimeToStringTransformer`](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToStringTransformer.php#L111).

Should we change this behavior there too ?